pkg/controller/logcollector/logcollector_controller.go

```diff
@@ -236,8 +281,9 @@ func (r *ReconcileLogCollector) Reconcile(ctx context.Context, request reconcile
 		return reconcile.Result{}, err
 	}
 
+	var exportLogs = utils.IsFeatureActive(license, common.ExportLogsFeature)
 	var s3Credential *render.S3Credential
-	if instance.Spec.AdditionalStores != nil {
+	if exportLogs && instance.Spec.AdditionalStores != nil {
```

Comment:
This one is a bit tricky since the component is installed, bu we limit the ability to export logs to a 3rd party system based on the type of license used.

Reply:
But the concept is basically "You asked for exporting to this external system, but your license doesn't allow for that" - so that should show up as a configuration error.

Reply:
Setting the status to degraded if you specify additional storage and do not have the correct license,

Reply:
I think we can remove the `exportLogs` part of the test here, because we've already checked it in the if statement above.